packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [43.0.0]
+
+### Added
+
+- `AccountTrackerController` now tracks balances of staked ETH for each account, under the state property `stakedBalance`. ([#4879](https://github.com/MetaMask/core/pull/4879))
+
+### Changed
+
+- Bump `@metamask/preferences-controller` from `^13.2.0` to `^13.3.0` ([#4909](https://github.com/MetaMask/core/pull/4909))
+- **BREAKING**: The polling input for`TokenListController` is now `{chainId: Hex}` instead of `{networkClientId: NetworkClientId}`. ([#4878](https://github.com/MetaMask/core/pull/4878))
+- **BREAKING**: The polling input for`TokenDetectionController` is now `{ chainIds: Hex[]; address: string; }` instead of `{ networkClientId: NetworkClientId; address: string; }`. ([#4894](https://github.com/MetaMask/core/pull/4894))
```

Comment:
Should we also include that `keyring-controller` was bumped? It also might be good to put the breaking changes at the top. Finally we will want to include the bump to `controller-utils`:

``` suggestion
- **BREAKING**: The polling input for`TokenListController` is now `{chainId: Hex}` instead of `{networkClientId: NetworkClientId}`. ([#4878](https://github.com/MetaMask/core/pull/4878))
- **BREAKING**: The polling input for`TokenDetectionController` is now `{ chainIds: Hex[]; address: string; }` instead of `{ networkClientId: NetworkClientId; address: string; }`. ([#4894](https://github.com/MetaMask/core/pull/4894))
- **BREAKING:** Bump `@metamask/keyring-controller` peer dependency from `^17.0.0` to `^18.0.0` ([#4195](https://github.com/MetaMask/core/pull/4195))
- Bump `@metamask/preferences-controller` from `^13.2.0` to `^13.3.0` ([#4909](https://github.com/MetaMask/core/pull/4909))
- - Bump `@metamask/controller-utils` from `^11.4.2` to `^11.5.0` ([#4195](https://github.com/MetaMask/core/pull/4195))
```